src/cascadia/TerminalControl/XamlUiaTextRange.cpp

```diff
@@ -100,9 +92,9 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         }
         else
         {
-            // We _need_ to return an empty object here.
+            // We _need_ to return XAML_E_NOT_SUPPORTED here.
             // Returning nullptr is an improper implementation of it being unsupported.

```

Comment:
Can you leave the comment explaining the history of you trying to return an empty object for future diagnostics?